### PR TITLE
feat(init): Add the possibility to pass options to init()

### PR DESCRIPTION
**Pull request by haadcode**
This PR will change the local() api so that options object can be passed to it.

PRing this again (original https://github.com/ipfs/js-ipfsd-ctl/pull/109).

### CODE REVIEW

**Comment by haadcode** [comment]
@dignifiedquire can I get a CR for this?

**Comment by dignifiedquire** [comment]
@haadcode can I get a test for that :)

File: test/index.spec.js [hunk 1]
```diff
@@ -345,8 +371,9 @@ describe('ipfs-api version', function () {
     })
   })
 
+  // NOTE: if you change ../src/, the hash will need to be changed
```
Comment (dignifiedquire) [inline test/index.spec.js]: this should be `if you change fixtures`

Comment (haadcode) [inline test/index.spec.js]: Fixed

**Comment by dignifiedquire** [comment]
Thank you @haadcode

**Comment by diasdavid** [comment]
🙌🏼

**Pull request merged**